android/jni/app/organicmaps/Framework.cpp

```diff
@@ -1210,6 +1210,8 @@ Java_app_organicmaps_Framework_nativeGetRouteFollowingInfo(JNIEnv * env, jclass)
 
   routing::FollowingInfo info;
   fr->GetRoutingManager().GetRouteFollowingInfo(info);
+  if (!info.IsValid())
+    return nullptr;
```

Comment:
Expect that all further code should process RoutingInfo = null ok, right?

Reply:
Yes. I returned back this line. It was removed in first Distnace PR